src/v2/view-builder/views/TerminalView.js

```diff
@@ -1,14 +1,32 @@
-import { createCallout } from 'okta';
+import { createCallout, loc } from 'okta';
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
+import BaseFooter from '../internals/BaseFooter';
+import {getBackToSignInLink} from '../utils/LinksUtil';
 
 const Body = BaseForm.extend({
   noButtonBar: true,
+  isTerminalForExpiredLink: false,
+
+  initialize () {
+    const messagesObjs = this.options.appState.get('messages');
+    this.isTerminalForExpiredLink = messagesObjs && Array.isArray(messagesObjs.value)
+      && messagesObjs.value.some(messagesObj => messagesObj.i18n?.key === 'idx.return.link.expired');
+    BaseForm.prototype.initialize.apply(this, arguments);
```

Comment:
We always call baseform Initialize method first and then override other stuff.. If you are unable to do that then you have to find some other way to do it.

Reply:
I will not need this since the advices from above is to use method instead of `isTerminalForExpiredLink ` variable.